src/services/api.ts

```diff
@@ -40,6 +40,8 @@ import {backend as BACKEND, metaBackend as META_BACKEND} from '../store';
 import {prepareSortValue} from '../utils/filters';
 import {parseMetaCluster} from './parsers/parseMetaCluster';
 import {parseMetaTenants} from './parsers/parseMetaTenants';
+import {settingsManager} from './settings';
+import {BINARY_DATA_IN_PLAIN_TEXT_DISPLAY} from '../lib';
```

Comment:
Could you please import not from '../lib' to prevent circular dependencies in the future?

Reply:
Fixed.